framework/CodeInterfaces/RELAP5/RELAPparser.py

```diff
@@ -19,35 +19,296 @@
+      for tripID, trip in self.inputTrips[deckNum]['variableTrips'].items():
+        found = False
+        for edit in alreadyAvailableEdits:
+          if trip['variable'] != 'time' and trip['component'] in edit['component'] and trip['variable'] in edit['component']:
+            found = True
+          elif trip['variable'] == 'time' and "timeof" in edit['variable'] and tripID in edit['component']:
+            found = True
+        # if not found and trip['variable']+"_"+trip['component'] not in addedVars:
+        if not found and trip['variable']+"_"+trip['component'] not in addedVars:
+          if len(availableMinorEditsCards) == 0: raise IOError("Number of minor edits reached already the upper bound of RELAP5. There are no available cards to monitor the trips!")
+          if trip['variable'] != 'time':
+            addedVars.append(trip['variable']+"_"+trip['component'])
+            self.deckLines[deckNum].append(str(availableMinorEditsCards.pop()) + " "+trip['variable']+" "+trip['component'] +"\n")
+          else:
+            addedVars.append("timeof"+"_"+tripID)
+            self.deckLines[deckNum].append(str(availableMinorEditsCards.pop()) + " "+"timeof"+" "+tripID +"\n")
+      if cntrVars is not None:
+        for cntrVar in cntrVars[deckNum]:
+          self.deckLines[deckNum].append(str(availableMinorEditsCards.pop()) + " "+"cntrlvar"+" "+cntrVar +"\n")
+      self.deckLines[deckNum].append("* END --  MINOR EDITS TRIPS ADDED BY RAVEN *\n")
+
+  def getTripsMinorEditsAndControlVars(self):
+    """
+      Method to store the Trips, Minor edits and Control Variables in RELAP5
+      @ In, None
+      @ Out, None
+    """
+    for deckNum in self.deckLines.keys():
+      self.lastTripLine[deckNum]     = -1
+      self.lastCntrLine[deckNum]     = -1
+      self.lastMinorEditLine[deckNum]= -1
+      for lineNum, line in enumerate(self.deckLines[deckNum]):
+        splitted = _splitRecordAndRemoveComments(line)
+        if len(splitted) > 0 and splitted[0].strip().isdigit():
+          isMinor = self.storeMinorEdit(deckNum,splitted)
+          isTrip  = self.storeTrip(deckNum,splitted)
+          isCntr  = self.storeControlVars(deckNum,splitted)
+          if isMinor:
+            self.lastMinorEditLine[deckNum] = lineNum+1
+          if isTrip :
+            self.lastTripLine[deckNum]      = lineNum+1
+          if isCntr :
+            self.lastCntrLine[deckNum]      = lineNum+1
+
+  def getMinorEditsInfo(self):
+    """
+    """
+    for deckNum in self.deckLines.keys():
+      for lineNum, line in enumerate(self.deckLines[deckNum]):
+        splitted = _splitRecordAndRemoveComments(line)
+        if len(splitted) > 0 and splitted[0].strip().isdigit():
+          isMinor = self.storeMinorEdit(deckNum, splitted)
+          if isMinor: self.lastMinorEditLine[deckNum] = lineNum+1
+
+  def storeControlVars(self,deckNum, splitted):
+    """
+    """
+    isCntr = False
+    if int(splitted[0]) == 20500000:
```

Comment:
should '20500000' be a variable in this interface?

Reply:
This card is hardcoded in RELAP (it is the card that flag the type of syntax for control variables, I can add it as a parser self constant variable but it will not ever be changed (if RELAP5 does not change it))